seqr/fixtures/users.json

```diff
@@ -29,7 +29,7 @@
     "fields": {
         "password": "pbkdf2_sha256$30000$y85kZgvhQ539$jrEC3L1IhCezUx3Itp+14w/T7U6u5XUxtpBZXKv8eh4=",
         "last_login": null,
-        "is_superuser": true,
+        "is_superuser": false,
```

Comment:
It is not clear what's "is_superuser" for?

Reply:
`is_superuser` used to override all permissions and grant access to everything. We were using it as a workaround for tests because you can grant superuser access here in the fixture unlike real collaborator/manager permissions and then the test user can access everything being tested. But in real life seqr no one is a superuser so its not really testing our authentication well, so I took out all the code that grants superusers special access and then I also changed the fixture data here since it is no longer used

Reply:
It makes sense.